RiotSharp/RiotApi.cs

```diff
@@ -35,7 +35,7 @@ public class RiotApi : IRiotApi
         private const string RunesRootUrl = "/api/lol/{0}/v1.4/summoner";
         private const string RunesUrl = "/{0}/runes";
 
-        private const string ChampionRootUrl = "/api/lol/{0}/v1.2/champion";
+        private const string ChampionRootUrl = "/lol/platform/v3/champions";
```

Comment:
Can you rename this to `ChampionsUrl` so it's consistent with the other v3 urls naming.